manifests/application/datamgmt.pp

```diff
@@ -38,17 +31,12 @@
     if $package_name == 'lightblue-data-mgmt-saml-auth' {
         include lightblue::authentication::saml
 
+        if $app_uri == undef {
```

Comment:
Ah, good catch. It is. I think there is a rare but easy enough to accommodate use case where someone might want to only allow cross-origin requests from a certain origin, and then protected access to the app at that origin via SAML. But it probably wouldn't take much to convince me to not support that use case. However, unless you would like otherwise, I'll go ahead and fix this so it at least works as is.